genrobots/main.go

```diff
@@ -104,6 +104,7 @@ func names(ctx context.Context, bucket *storage.BucketHandle) (map[string]struct
 		if !strings.HasPrefix(obj.Name, "dotnet") &&
 			!strings.HasPrefix(obj.Name, "go") &&
 			!strings.HasPrefix(obj.Name, "nodejs") &&
+			!strings.HasPrefix(obj.Name, "java") &&
```

Comment:
Totally minor nit: would you mind moving this up one line to keep this in alphabetical order?